notebooks/olmocr-pdf-vlm/olmocr-pdf-vlm.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        "tokenizers&lt;0.21,&gt;=0.20",
</pre><p>let transformers decide which tokenizers are compatible (as I know  it is default version for now, but something can changed in future <a href="https://github.com/huggingface/transformers/blob/main/setup.py#L188" target="_blank">https://github.com/huggingface/transformers/blob/main/setup.py#L188</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>